de.dlr.sc.virsat.excel/src/de/dlr/sc/virsat/excel/fault/IFaultType.java

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.excel.fault;
+
+/**
+ * This interface is for all possible faults which can happen during import
+ */
+public interface IFaultType {
+	

```

Comment:
Let's add a short comment explaining why this interface is empty

Reply:
@andriikovalov-dlr I added something as an explanation, but I am not sure if it is oky. Could you please check it?